src/library/scala/concurrent/Done.scala

```diff
@@ -0,0 +1,36 @@
+/*                     __                                               *\
+**     ________ ___   / /  ___     Scala API                            **
+**    / __/ __// _ | / /  / _ |    (c) 2003-2013, LAMP/EPFL             **
+**  __\ \/ /__/ __ |/ /__/ __ |    http://scala-lang.org/               **
+** /____/\___/_/ |_/____/_/ | |                                         **
+**                          |/                                          **
+\*                                                                      */
+
+package scala.concurrent
```

Comment:
Seems a bit weird to put this in concurrent, wouldn't it be more fitting in util?
(I understand that "Done" is rather indicative of progress)

I could be convinced either way.